rest/src/commonMain/kotlin/service/EntitlementService.kt

```diff
@@ -0,0 +1,57 @@
+package dev.kord.rest.service
+
+import dev.kord.common.entity.DiscordEntitlement
+import dev.kord.common.entity.Snowflake
+import dev.kord.rest.json.request.TestEntitlementCreateRequest
+import dev.kord.rest.request.RequestHandler
+import dev.kord.rest.route.Position
+import dev.kord.rest.route.Route
+
+public class EntitlementService(handler: RequestHandler) : RestService(handler) {
+
+    public suspend fun getEntitlements(
+        applicationId: Snowflake,
+        position: Position? = null,
+        limit: Int? = null,
+        guildId: Snowflake? = null,
+        userId: Snowflake? = null,
+        skuIDs: List<Snowflake>? = null,
+        excludeEnded: Boolean? = null,
```

Comment:
I thought so too but saw methods with similar signatures so I decided against it, good to know!

Reply:
Should EntitySupplier#getEntitlements also receive this change?

Reply:
> I thought so too but saw methods with similar signatures so I decided against it, good to know!

i think in this case it's better to pass the request, there are a lot of params otherwise. i think a builder like `AuditLogGetRequestBuilder` would be good too.

Reply:
> Should EntitySupplier#getEntitlements also receive this change? (i assume not but just checking)

will look at this, when looking at `core`